test/system/reports_test.rb

```diff
@@ -3,45 +3,101 @@
 require 'application_system_test_case'
 
 class ReportsTest < ApplicationSystemTestCase
-  setup do
-    @report = reports(:one)
+  def setup
+    @user = FactoryBot.create(:user)
+    login_as(@user.email, @user.password)
   end
 
-  test 'visiting the index' do
+  test 'elements on report index page' do
+    report = FactoryBot.create(:report, user_id: @user.id, created_at: Time.zone.parse('2023-08-30 12:00:00'))
+
     visit reports_url
-    assert_selector 'h1', text: 'Reports'
+    assert_selector 'h1', text: '日報の一覧'
+    within 'div.index-item' do
+      within "div#report_#{report.id}" do
+        assert_selector 'p', text: report.title
+        assert_selector 'p', text: report.content
+        assert_selector 'p', text: report.user.name
+        assert_selector 'p', text: '2023/08/30'
+      end
+      assert_selector 'a', text: 'この日報を表示'
+    end
+    assert_selector 'a', text: '日報の新規作成'
   end
 
-  test 'should create report' do
-    visit reports_url
-    click_on 'New report'
+  test 'elements on report show page' do
+    report = FactoryBot.create(:report, user_id: @user.id, created_at: Time.zone.parse('2023-08-30 12:00:00'))
+
+    visit report_url report
+    assert_selector 'h1', text: '日報の詳細'
+    within "div#report_#{report.id}" do
+      assert_selector 'p', text: report.title
+      assert_selector 'p', text: report.content
+      assert_selector 'p', text: report.user.name
+      assert_selector 'p', text: '2023/08/30'
+    end
+    assert_selector 'a', text: 'この日報を編集'
+    assert_selector 'a', text: '日報の一覧に戻る'
+    assert_selector 'button', text: 'この日報を削除'
+  end
+
+  test 'create a new report with proper inputs' do
+    assert_current_path '/books'
+
+    visit new_report_url
+    fill_in 'report[title]', with: 'チェリー本を読んだ'
```

Comment:
```suggestion
    fill_in 'タイトル', with: 'チェリー本を読んだ'
```

のように画面に見えているラベル名を使った方が可読性が上がります。

Reply:
そっちの方がいいですね！